src/Paysera/WalletApi/Entity/Client.php

```diff
@@ -55,6 +55,11 @@ class Paysera_WalletApi_Entity_Client
      */
     protected $credentials;
 
+    /**
+     * @var Paysera_WalletApi_Entity_ClientPermissionsToWallet[]
+     */
+    protected $permissionsToWallets = [];
```

Comment:
we should set it in the constructor, not where we name variable.

Reply:
Done